app/controllers/blazer/queries_controller.rb

```diff
@@ -369,6 +371,15 @@ def get_assignees
       []
     end
 
+    def get_assignees_with_email
+      return [] unless Blazer.settings.key?('assignees_with_email')
+      Blazer::RunStatement.new.perform(@data_source, Blazer.settings['assignees_with_email'], {}).rows.map do |row|
+        [row.first, "#{row.second.to_s.titleize} - #{row.last}"]
```

Comment:
https://github.com/ThanhKhoaIT/blazer/pull/6/files#diff-1a56aa52ca7b3883dba708b084406cb2eb99987df2413812dba2856e8add86f1R368
can change to 2 case
```ruby
case row.size
when 2
  [row.first, row.last.to_s.titleize]
when 3
  [row.first, "#{row.second.to_s.titleize} - #{row.last}"]
else
  [row.first, row.first]
end
```